Change "Guidelines" links on Usability.gov

nick-mon1 opened the pull request:
Update links from `https://guidelines.usability.gov/` to `https://www.usability.gov/sites/default/files/documents/guidelines_book.pdf`.

This affects the sidebar navigation button and body links.

<img width="399" alt="Screen_Shot_2022-07-26_at_2 05 03_PM" src="https://user-images.githubusercontent.com/104778659/182651119-75961402-28cb-4888-bcc3-f64c476c2442.png">

nick-mon1 committed:
updated link with new href

mejiaj reviewed (commented):
Alternatively, we could set this value as a custom variable in `_config.yml` and then update all templates once. In the future it would be easier to update (not sure if it would).



---

Some additional instances of that link. Do we need to update these too?

https://github.com/GSA/digitalgov-usability/blob/306e78b9aba2010e46ca08ca56e09116b9068580/get-involved/blog/2005/08/basics-for-optimal-usability.html#L260

https://github.com/GSA/digitalgov-usability/blob/306e78b9aba2010e46ca08ca56e09116b9068580/get-involved/blog/2013/welcome-to-relaunched-site.html#L297-L298

https://github.com/GSA/digitalgov-usability/blob/306e78b9aba2010e46ca08ca56e09116b9068580/get-involved/blog/2013/07/welcome-to-relaunched-site.html#L299-L300

https://github.com/GSA/digitalgov-usability/blob/306e78b9aba2010e46ca08ca56e09116b9068580/node/75.html#L258

https://github.com/GSA/digitalgov-usability/blob/306e78b9aba2010e46ca08ca56e09116b9068580/node/1785.html#L298

nick-mon1 committed:
updated link on 5 files

nick-mon1 commented:
Thanks @mejiaj for catching those missed links!

ToniBonittoGSA committed:
Update basics-for-optimal-usability.html

added file info `(PDF, 21.9 MB, 292 pages)` on links for https://www.usability.gov/sites/default/files/documents/guidelines_book.pdf

ToniBonittoGSA committed:
Update welcome-to-relaunched-site.html

added file info `(PDF, 21.9 MB, 292 pages)` on links for https://www.usability.gov/sites/default/files/documents/guidelines_book.pdf

ToniBonittoGSA committed:
Update welcome-to-relaunched-site.html

added file info `(PDF, 21.9 MB, 292 pages)` on links for https://www.usability.gov/sites/default/files/documents/guidelines_book.pdf

ToniBonittoGSA committed:
Update 75.html

added file info `(PDF, 21.9 MB, 292 pages)` on links for https://www.usability.gov/sites/default/files/documents/guidelines_book.pdf

ToniBonittoGSA committed:
Update 1785.html

added file info `(PDF, 21.9 MB, 292 pages)` on links for https://www.usability.gov/sites/default/files/documents/guidelines_book.pdf
